app/views/lists/_list.html.erb

```diff
@@ -0,0 +1,5 @@
+<li>
+    <%= link_to list.id, list_path %>
+    <%= link_to list.name, list_path %>
+    <%= link_to list.description, list_path %>
+</li>
```

Comment:
Not important for now, but you should to configure your vs code to make so that every file end in a new line and your tab should be 2 spaces.